### DISCUSSION TITLE
Entry level permissions

### DISCUSSION THREAD

**Discussion by korvin87** [craftcms/cms#8676] [Ideas]
### Description
We would love to have the following function: only members of a certain usergroup are allowed to view, edit and delete a subset of entries within one certain section in the backend. Members that do not have this usergroup should not be able to view, edit and delete the entries from this subset . Also we do need several usergroups, each needs to have access (view, edit and delete) to its own subset of entries within a section
For the time being we do not see such a strict separation between different users. Is there any chance to overcome this?

### Additional info

- Craft version: 3.1.26
- PHP version: 7.2
- Database driver & version: MySQL
- Plugins & versions: none

**Comment by mtnorthrop** [comment]
Until entry-level permissions are added to CraftCMS, the [Isolate plugin](https://plugins.craftcms.com/isolate) might help people who are looking for this feature. It allows entry-level permissions to be set on a **per user** basis.

**Comment by aaronbushnell** [comment]
FWIW—with all the new slide-out functionality in Craft, and the [entrification](https://craftcms.com/blog/entrification) plan for Craft 5+ we aren't planning to update Isolate to be Craft 5+ compatible. There's just too many scenarios to account for (including things like multisite, as well).

Would love to something comparable baked into Craft natively at some point, but I'm sure the Pixel and Tonic folks have their hands full [with all the other planned items](https://craftcms.com/roadmap)!

**Comment by iuscare** [comment]
Might be duplicated of https://github.com/craftcms/cms/discussions/9156
